src/Nethermind/Nethermind.Mev.Test/MevRpcModuleTests.cs

```diff
@@ -281,19 +308,222 @@ public async Task Should_include_bundle_transactions_uninterrupted_in_order_from
             GetHashes(chain.BlockTree.Head!.Transactions).Should().Equal(GetHashes(new Transaction[] { set1, set2, set3, tx4, tx5 }));
         }
 
-        private void SuccessfullySendBundle(TestMevRpcBlockchain chain, int blockNumber, params Transaction[] txs)
+        [Test]
+        public async Task Should_merge_disjoint_bundles()
         {
-            byte[][] bundleBytes = txs.Select(t => Rlp.Encode(t).Bytes).ToArray();
-            ResultWrapper<bool> resultOfBundle = chain.MevRpcModule.eth_sendBundle(bundleBytes, blockNumber);
-            resultOfBundle.GetResult().ResultType.Should().NotBe(ResultType.Failure);
-            resultOfBundle.GetData().Should().Be(true);
+            var chain = await CreateChain(SelectorType.V2, 3);
+            chain.GasLimitCalculator.GasLimit = 10_000_000;
+
+            Transaction tx1 = Build.A.Transaction.WithGasLimit(GasCostOf.Transaction).WithGasPrice(150ul).SignedAndResolved(TestItem.PrivateKeyA).TestObject;
+            Transaction tx2 = Build.A.Transaction.WithGasLimit(GasCostOf.Transaction).WithGasPrice(100ul).SignedAndResolved(TestItem.PrivateKeyB).TestObject;
+            Transaction tx3 = Build.A.Transaction.WithGasLimit(GasCostOf.Transaction).WithGasPrice(50ul).SignedAndResolved(TestItem.PrivateKeyC).TestObject;
```

Comment:
Generally for this and later, you can name transactions better for intention like:
bundle1tx1, bundle2tx2
bundle3tx1LowGas ect.
memPoolTx2HighGas